### DISCUSSION TITLE
Palico/Palamute editing?

### DISCUSSION THREAD

**Discussion by sneedfeedseed** [Glazelf/RiseHax#15] [Ideas]
Hello is this project still ongoing? If not sorry I didnt know... but if it still is here’s an idea you can include!
Palico/Palamute editting
-Edit Skills
-Edit Levels
-Edit Moves
-Edit Support Type
among other things, if you read this thank you for your time! I’m looking forward to this project.

**Comment by Glazelf** [comment]
It's still ongoing, yes, though progress is going kind of slow with the frequent updates, Noexs being stupid and me being busy in real life!
This looks pretty complicated and big, so it might take a while, but it is a cool idea! It would be cool to test out some impossible palico skill combinations... :o

**Reply by sneedfeedseed** [reply]
thanks for the reply! good to know the situation, please take your time no rush at all! and yeah testing out different palico moves combinations does sounds like fun!
